pkg/resource/v1/certconfig/desired_test.go

```diff
@@ -1 +1,91 @@
 package certconfig
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned/fake"
+	"github.com/giantswarm/micrologger"
+	clientgofake "k8s.io/client-go/kubernetes/fake"
+
+	"github.com/giantswarm/cluster-operator/pkg/cluster"
+	"github.com/giantswarm/cluster-operator/pkg/resource/v1/certconfig/key"
+)
+
+func Test_GetDesiredState_Returns_CertConfig_For_All_Managed_Certs(t *testing.T) {
+	logger, err := micrologger.New(micrologger.DefaultConfig())
```

Comment:
There isn't DefaultConfig in new logger package. Please start using `micrologger.New(micrologger.Config{})` in the new code.

In tests we usually use `microloggertest.New()` - it doesn't return error.

Reply:
Good catch. I'll fix this here and clean up all other occasions in separate PR.

Reply:
Cool